Scripts/SaneDocReport/SaneDocReport.py

```diff
@@ -10,13 +13,20 @@
 try:
     sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
         "utf-8")
+    orientation = demisto.args().get('orientation', 'portrait').encode(
+        "utf-8")
```

Comment:
i think the codec should be provided as an argument with default value of `utf-8`
what do you think ?

Reply:
I don't understand, did you mean encoding? and if so again, what do you mean?